src/Monolog/Handler/StreamHandler.php

```diff
@@ -32,13 +32,12 @@ class StreamHandler extends AbstractProcessingHandler
     private $dirCreated;
 
     /**
-     * @param resource|string $stream
+     * @param resource|string $stream         If a missing path can't be created, an Exception will be thrown on first write
```

Comment:
Should say UnexpectedValueException not Exception?

Reply:
The latest update fixes it.